src/components/Pricing/data.ts

```diff
@@ -0,0 +1,499 @@
+    },
+    comprehensiveApi: {
+        label: 'Comprehensive API',
+        description: 'A secure, robust GraphQL API for your repository and code data.',
+    },
+}
+
+const SECURITY_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ssoSaml: {
+        label: 'SSO/SAML',
+        description: 'Single sign-on user authentication with SAML, OAuth, OpenID Connect, and HTTP auth proxy.',
+    },
+    securityRoles: {
+        label: 'User and admin roles',
+        description:
+            'Allow only certain users (site admins) to view and edit site configuration and repository/code host credentials.',
+    },
+    repoPerms: {
+        label: 'Standard repository permissions',
+        description:
+            'Apply the repository permissions from your code host to restrict which repositories a user can search and browse.',
+    },
+    customPerms: {
+        label: 'Custom repository permissions',
+        description:
+            'Apply the repository permissions from your code host or use the explicit permissions API to restrict which repositories a user can search and browse.',
+    },
+    privateInstance: {
+        label: 'Private instance access',
+        description: 'Your instance is only accessible from allowlisted IPs. ',
+    },
+    analytics: {
+        label: 'In-product analytics',
+        description: 'Understand user engagement and calculate the value of using Sourcegraph.',
+    },
+}
+
+const DEPLOYMENT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    cloudDeployment: {
+        label: 'Secure and single-tenant Cloud deployment',
+        description: 'Your own dedicated, single-tenant Sourcegraph instance.',
+    },
+    selfDeployment: {
+        label: 'Self-hosted deployment',
+        description: 'Deploy with Docker, Docker Compose, or Kubernetes on your own infrastructure.',
+    },
+}
+
+export const SPOTLIGHT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ...CIP_FEATURE_INFO,
+
+    // Code host integrations
+    cloudHosts: {
+        label: 'Unlimited standard Cloud code hosts',
+        description: 'Integrate with GitHub.com, GitLab.com, and Bitbucket Cloud.',
+    },
+    unlimitedCodeHosts: {
+        label: 'Unlimited code hosts',
+        description: 'Search, understand, fix, and automate across multiple code hosts.',
+    },
+    selfHostedCodeHosts: {
+        label: 'Connect to self-hosted code hosts',
+        description:
+            'Integrate with Github Enterprise Self-hosted, GitLab Self-hosted, Bitbucket Server/Data Center, and Perforce.',
+    },
+    enterpriseOnlyCodeHosts: {
+        label: 'Connect to enterprise-only code hosts',
+        description: 'Integrate with any Git-based code host via src-srv-git.',
+    },
+    privateCodeHosts: {
+        label: 'Connect to private code hosts',
+        description: 'Integrate with supported code hosts stored behind a firewall (only available for self-hosted deployments).',
```

Comment:
* Inconsistent usage of "integrate" vs. "connect" for the verb describing Sourcegraph's interaction with code hosts.
* A code host can't be "stored" behind a firewall. "Stored" is not the right verb (it would be located, deployed, etc.)
* And "firewall" is not the right term. I think people will know what we mean if we say "firewall", but I believe the more technically precise term would be "on a private network" or something like that.

Reply:
CC: @abschu for copy confirmation

Reply:
@sqs - we use "Connect" colloquially in the docs, but I see how it's confusing. Let's stick with integrate.

Integrate with self-hosted code hosts
Integrate with enterprise-only code hosts
Integrate with private code hosts

Private code host tooltip: Integrate with supported code hosts deployed on a private network (only available for self-hosted deployments).